config.yml

```diff
@@ -21,7 +21,7 @@ defaults:
       type: 'oauth'
 
   webhook:
-    secret: !env WEBHOOK_SECRET
+    secret: !env:list WEBHOOK_SECRETS
```

Comment:
I made it singular, so that the existing vars could work. In the code though the variable that holds the array is still plural.